cache/sources_stats.go

```diff
@@ -0,0 +1,116 @@
+package cache
+
+import (
+	"database/sql"
+
+	"github.com/Bnei-Baruch/sqlboiler/queries"
+	"github.com/pkg/errors"
+	"gopkg.in/volatiletech/null.v6"
+
+	"github.com/Bnei-Baruch/archive-backend/mdb"
+)
+
+type SourcesStatsCache interface {
+	Refresh() error
+	GetChildren(rootUIDs []string) ([]string, []int64)
+	GetHistogram() ClassByTypeStats
+}
+
+type SourcesStatsCacheImpl struct {
+	mdb  *sql.DB
+	tree *StatsTree
+}
+
+func NewSourcesStatsCacheImpl(mdb *sql.DB) SourcesStatsCache {
+	stats := new(SourcesStatsCacheImpl)
+	stats.mdb = mdb
+	return stats
+}
+
+func (s *SourcesStatsCacheImpl) Refresh() error {
+	err := s.load()
+	return errors.Wrap(err, "Load tags and sources stats.")
```

Comment:
rephrase message. we only load sources here not tags.

Reply:
fix